### ISSUE TITLE
"Dim display when inactive" doesn't appear to do anything

### ISSUE THREAD

**Issue by cassidyjames** [elementary/switchboard-plug-power#95]
It doesn't appear to change the display backlight after any amount of time.

**Comment by danrabbit** [comment]
Yeah after some discussion it doesn't sound like this feature really has any purpose. You might as well turn the display off if you really care about power saving here and this setting may not just do anything at all for many settings
